pkg/rtc/transport.go

```diff
@@ -1609,17 +1606,25 @@ func (t *PCTransport) handleRemoteICECandidate(e *event) error {
 	return nil
 }
 
-func (t *PCTransport) handleLogICECandidates(e *event) error {
+func (t *PCTransport) handleLogICECandidates(_ *event) error {
 	lc := t.allowedLocalCandidates.Get()
 	rc := t.allowedRemoteCandidates.Get()
+	var fields []interface{}
 	if len(lc) != 0 || len(rc) != 0 {
-		t.params.Logger.Infow(
-			"ice candidates",
+		fields = append(fields,
 			"lc", lc,
 			"rc", rc,
-			"lc (filtered)", t.filteredLocalCandidates.Get(),
-			"rc (filtered)", t.filteredRemoteCandidates.Get(),
+			"lc_filtered", t.filteredLocalCandidates.Get(),
+			"rc_filtered", t.filteredRemoteCandidates.Get(),
```

Comment:
spaces in fields are harder to parse. temporary for now, I'll be changing how we handle ice candidates in a separate PR